slim.js.org

eavichay opened the pull request:
- There is reasonable content on the page (see: [No Content](https://github.com/js-org/dns.js.org/wiki/No-Content))
- I have read and accepted the [ToS](http://dns.js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```slim.js.org``` to make it work.
I'll merge after we have "contact".

indus commented:
looks like I merged to early. Just add the CNAME so I don´t have to revert the PR

eavichay commented:
done. Added to root + docs folder.


On Tue, Mar 7, 2017 at 5:15 PM, Stefan Keim <notifications@github.com>
wrote:

> I've already added your requested subdomain to JS.ORGs zonefile. But you
> have to (re-)add a file named CNAME to your branch with the single line
> slim.js.org to make it work.
> I'll merge after we have "contact".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/dns.js.org/pull/1110#issuecomment-284750208>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCMK1LBoeYOiSNttrHBn6wjkqZNqQgHks5rjXSmgaJpZM4MVO6d>
> .
>



-- 
-Avichay

eavichay commented:
done.,

On Wed, Mar 8, 2017 at 8:29 AM, Stefan Keim <notifications@github.com>
wrote:

> looks like I merged to early. Just add the CNAME so I don´t have to revert
> the PR
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/dns.js.org/pull/1110#issuecomment-284959128>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCMK4j4Mskz4tlZMCYyeLTl6411BVv3ks5rjkrSgaJpZM4MVO6d>
> .
>



-- 
-Avichay

indus commented:
Thanks"
